src/Symfony/Component/HttpFoundation/Session/Storage/Handler/RedisSessionHandler.php

```diff
@@ -63,7 +63,15 @@ public function __construct($redis, array $options = [])
 
         $this->redis = $redis;
         $this->prefix = $options['prefix'] ?? 'sf_s';
-        $this->ttl = $options['ttl'] ?? (int) ini_get('session.gc_maxlifetime');
+        $this->ttl = $options['ttl'] ?? null;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    private function getTtl(): int
+    {
+        return $this->ttl ?? ini_get('session.gc_maxlifetime');
```

Comment:
let's inline this, no need for the method
we'd rther use `(int) ($options['ttl'] ?? ini_get('session.gc_maxlifetime'))` btw

Reply:
Are you reason with the type (int), and if you prefer inline, i dont have any problem with.